server.js

```diff
@@ -4,12 +4,49 @@ const path = require('path');
 
 const app = express();
 
-// view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'hbs');
 app.use(express.static(path.join(__dirname, 'public')));
 
-// add your routes here :)
+app.get('/:code', (req, res) => {
+  const code = req.params.code;
+  const promise1 = fetch('http://localhost:3000/code/' + code)
+  .then(response => response.json());
+  const promise2 = fetch('http://numbersapi.com/' + code)
+  .then(response => response.text());
+
+  Promise.all([promise1, promise2])
+  .then((arr) => {
+    res.render('code', { 
+      'code': arr[0].code, 
+      'phrase': arr[0].phrase,
+      'description': arr[0].description, 
```

Comment:
Rather than referencing `arr[0]` repeatedly. It is best to set it to a variable. It isn't critical for this case, but it is a good habit to get into, especially when cacheing objects.(see tip 1 [here](http://www.javascriptkit.com/javatutors/efficientjs.shtml))

Example
```js
const promiseOneRet = arr[0];

const someObj = {
    one: promiseOneRet.phase,
    two: promiseOneRet.description
}
```